Mage/src/main/java/mage/abilities/hint/HintUtils.java

```diff
@@ -16,6 +16,7 @@ public class HintUtils {
     // icons changes to real files on client side (see mana icons replacement)
     public static final String HINT_ICON_GOOD = "ICON_GOOD";
     public static final String HINT_ICON_BAD = "ICON_BAD";
+    public static final String HINT_ICON_EVASION = "ICON_RESTRICT"; //TODO: add a separate icon. A cape maybe?
```

Comment:
I'll leave more GUI improvment for later, like having another icon for the hint line and/or a generic evasion icon in the left icon panel.